providers/ethereum-provider/src/index.ts

```diff
@@ -1,3 +1,5 @@
 import { EthereumProvider as Provider } from "./EthereumProvider";
 export const EthereumProvider = Provider;
+export type { RpcEvent, RpcMethod } from "./EthereumProvider";
+export { OPTIONAL_METHODS, REQUIRED_EVENTS, REQUIRED_METHODS, OPTIONAL_EVENTS } from "./constants";
```

Comment:
if you're exporting everything from `./constants/rpc` would it be better to export all? 
When we add new constants, we wouldn't have to update the `index.ts` as well 
```suggestion
export * from "./constants/rpc";
```